### DISCUSSION TITLE
In the general settings, there is no GitHub Pages section.

### DISCUSSION THREAD

**Discussion by achmadaffan** [community/community#64483] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

thank you for answer

**Comment by arafajs** [comment]
It's "Pages".

**Comment by zainmujahid** [comment]
Hi, you can find the option here as shown under general settings:

![image](https://github.com/community/community/assets/34394344/dff2c1ae-64f9-4868-958e-02001f8a1d86)

**Reply by achmadaffan** [reply]
![github pages](https://github.com/community/community/assets/12270499/8803587a-c84d-43eb-992d-b810c96f7a9a)
### on my pages is not like that. you know why?

**Reply by Nhalrath** [reply]
This page is already telling you what to do.

Are you trying to host a page from a private repo? If that's the case, then you need to make it public or get a pro subscription.

**Reply by achmadaffan** [reply]
thank you sir

**Comment by RONIE70** [comment]
The UI has changed a bit since that video was made. There should be a "Pages" entry in the menu on the left of the repository settings page, click that.

![image](https://github.com/community/community/assets/72896165/1a8458cb-8d26-4f0c-acb6-7fcd27797ab0)

**Reply by achmadaffan** [reply]
Thank you sir
I have found the solution to fix this
